### DISCUSSION TITLE
Automatic suspend server due to missing payment

### DISCUSSION THREAD

**Discussion by mrdadone** [Paymenter/Paymenter#433] [Ideas]
add on settings "Days after wich an upaid invoice will be suspended" , like with the remove one.

**Comment by GomorrhaDev** [comment]
Why would you add that, I don't understand what advantage it has

**Comment by mrdadone** [comment]
Well some people might not see the invoice , and they would wake up with a deleted server, suspending it will "warn" about the missing payment

**Comment by purejosh** [comment]
Oh, you mean to keep the server active, but not suspend it, if they're late? That's quite forgiving. I'm not sure that feature has validity, but maybe some people will want it.
